plugin/src/main/java/org/opensearch/ml/action/connector/DeleteConnectorTransportAction.java

```diff
@@ -53,11 +56,13 @@ public DeleteConnectorTransportAction(
         TransportService transportService,
         ActionFilters actionFilters,
         Client client,
```

Comment:
Should we remove this ?

Reply:
No, because we still need it to access other key things like Thread Pools, Thread Context (and User), etc. as well as potentially any `client.execute()` calls to other Transport Actions.   The `SdkClient` is only abstracting out the data access (CRUD-S) features of the `NodeClient`.